debian/changelog

```diff
@@ -1,32 +1,18 @@
-bluez (5.53+ubports5+really5.53-0ubports3) xenial; urgency=medium
+bluez (5.42+ubports6+really5.41-0ubports1) xenial; urgency=medium
```

Comment:
```suggestion
bluez (5.53+ubports6+really5.41-0ubports1) xenial; urgency=medium
```

The `5.53` in the front has to stay the same, as otherwise the final version in the apt archive would be lower, which would cause this not to land in the OTA image.